exercises/saddle-points/src/example.h

```diff
@@ -5,15 +5,16 @@
 #include <stdint.h>
 
 typedef struct {
-   size_t row, column;
+   size_t row;
+   size_t column;
 } saddle_point_t;
 
 typedef struct {
    size_t count;
-   saddle_point_t points[];
+   saddle_point_t *points;
```

Comment:
The aim was to specifically introduce flexible array members rather than just one more pointer exercise.

Reply:
I am not a C expert, but it seems that if we use flexible array, we can't use memory allocation on it. Instead, we should do a single allocation with size of struct + size of array, just as the code it is right now. If we choose to keep flexible array declaration, which seems to be easier and simpler, we can discard this PR.
I don't see how the tests can handle all cases. If a solution declares points as pointer and we don't free it, there will be a memory leak, but if the user declares points as a flexible array and we try to free it, we will get an invalid pointer error.
Maybe we can provide a boilerplate in header file.

Reply:
Maybe I need to look at the student solutions to date to fully understand the issue. I might get to this tomorrow evening (I will have to do submissions for all the other exercises up to that point to see them, so it won't happen this evening as it's late in my timezone).

Reply:
@wolf99 I think there's a way to declare yourself a maintainer/mentor so that you can view all exercises and all submitted solutions for a track. I have done it on C, but don't recall how.